src/workflows/airqo_etl_utils/bigquery_api.py

```
@@ -1002,21 +1015,7 @@ def fetch_satellite_readings(
-        """saves the dataframes to the bigquery tables"""
-        credentials = service_account.Credentials.from_service_account_file(
-            configuration.GOOGLE_APPLICATION_CREDENTIALS
-        )
-        data.to_gbq(
-            destination_table=f"{table}",
-            project_id=configuration.GOOGLE_CLOUD_PROJECT_ID,
-            if_exists="append",
-            credentials=credentials,
-        )
-        print(" data saved to bigquery")
+            logger.info(f"Error fetching data from bigquery", {e})
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)